pages/common/git-bundle.md

```diff
@@ -0,0 +1,32 @@
+# git bundle
+
+> Package objects and references in an archive.
```

Comment:
Very minor really, but perhaps this will read better?
```suggestion
> Package objects and references into an archive.
```
....unless this changes the meaning to be incorrect?

Reply:
the meaning in your suggestion is correct 👍